commercial/app/views/contentapi/items.scala.html

```diff
@@ -28,7 +28,8 @@ <h3 class="commercial__title">
                     </a>
                 </div>
                 <div class="commercial__body">
-                    <ul class="u-unstyled l-row l-row--cols-4 l-row--layout-m fc-slice">
+                    <div class="fc-slice-wrapper">
```

Comment:
It's a shame we have to add a `DIV` for that. There are so many elements in the DOM, consuming memory for the sake of presentation; the opportunities to get rid of that are legions. Could `DIV.commercial__body.fc-slice-wrapper` work?